### PR TITLE
Functionality for message board

### LINKED ISSUE #1384: Country Home/Message board
- [x] Add UI
- [x] Integrate message board actions

### PR DESCRIPTION
**Pull request by sorja**
Resolves #1384 

Todo:
- New message count

### CODE REVIEW

File: client/pages/AssessmentHome/FraHome/CountryMessageBoard/MessageBoard.tsx [hunk 1]
```diff
@@ -37,11 +43,22 @@ const MessageBoard = () => {
                 type="button"
                 className="btn-secondary landing__user-btn-message"
                 onClick={() => {
-                  // Todo
-                  // if (!countryMessageBoardOpened) {
-                  //   dispatch(openCountryMessageBoard())
-                  //   dispatch(closeChat())
-                  // }
+                  // TODO
+                  const countryMessageBoardOpened = false
```
Comment (minotogna) [inline client/pages/AssessmentHome/FraHome/CountryMessageBoard/MessageBoard.tsx:47]: remove the todo and add the logic if it's really necessary

File: client/pages/Assessment/Assessment.tsx [hunk 2]
```diff
@@ -90,6 +91,7 @@ const Assessment: React.FC = () => {
           )}
         />
       </Switch>
+      <MessageCenter />
```
Comment (minotogna) [inline client/pages/Assessment/Assessment.tsx:94]: the message center needs all query params from url. adding it here will brake the review .
it's needed to think to another solution

File: client/pages/AssessmentHome/FraHome/CountryMessageBoard/MessageBoardUsers.tsx [hunk 3]
```diff
@@ -32,28 +35,41 @@ const MessageBoardUsers = () => {
       </div>
       {users.map((_user) => (
         <div key={_user.id} className="landing__user-outer-container">
+          {/* TODO */}
```
Comment (minotogna) [inline client/pages/AssessmentHome/FraHome/CountryMessageBoard/MessageBoardUsers.tsx:38]: what;s this todo?

**Review by minotogna — CHANGES_REQUESTED** [review]

File: client/pages/AssessmentHome/FraHome/CountryMessageBoard/MessageBoardUsers.tsx (hunk 3 above)
Comment (sorja) [reply client/pages/AssessmentHome/FraHome/CountryMessageBoard/MessageBoardUsers.tsx:38]: related to user.active

File: client/pages/Assessment/Assessment.tsx (hunk 2 above)
Comment (sorja) [reply client/pages/Assessment/Assessment.tsx:94]: #1406

Comment (minotogna) [reply client/pages/Assessment/Assessment.tsx:94]: Find the solution . What else should you do?
Add the message center to Fra home?

Comment (sorja) [reply client/pages/Assessment/Assessment.tsx:94]: After further testing, it is working as expected in `Assessment.tsx` as verified with @rfontanarosa

**Review by minotogna — APPROVED** [review]

**Pull request merged**